jenkins/Jenkinsfile

```diff
@@ -268,7 +268,7 @@ pipeline {
             cd jenkins/dataproc-custom-images/dataproc-custom-images
             python generate_custom_image.py \
               --image-name "$IMAGE_BASE_NAME-$imageID" \
-              --dataproc-version "1.2.79-debian9" \
+              --dataproc-version "1.4.15-debian9" \
```

Comment:
shall we upgrade python version as well?

@rtitle shall we make the script to support both old and new versions?

Reply:
Btw, it would be good to revisit this code when upgrading Hail: https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/HttpGoogleDataprocDAO.scala#L329-L350

I bet we could remove the `"spark:spark.yarn.am.memory"` setting. Also we currently set the image to `1.2-deb9` if the custom image is empty. Not sure how to handle this… should we just require that a custom image is used going forward?

Reply:
I think it makes sense to always use custom dataproc image going forward